src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/epics/templates.epics.js

```diff
@@ -60,38 +60,35 @@ export const updateTemplateEpic = (action$: InputObservable, store: ReduxStore)
                 programId,
                 listId,
             } = action.payload;
-
+            const {
+                id,
+                name,
+                externalAccess,
+                publicAccess,
+                user,
+                userGroupAccesses,
+                userAccesses,
+            } = template;
             const eventFilterData = {
-                name: template.name,
+                name,
                 program: programId,
                 eventQueryCriteria,
+                externalAccess,
+                publicAccess,
+                user,
+                userGroupAccesses,
+                userAccesses,
             };
 
             const api = getApi();
 
             const requestPromise = api
-                .update(`eventFilters/${template.id}`, eventFilterData)
-                .then(() => api
-                    .post(`sharing?type=eventFilter&id=${template.id}`, {
-                        object: {
-                            publicAccess: '--------',
-                            externalAccess: false,
-                        },
-                    })
-                    .catch((error) => {
-                        log.error(
-                            errorCreator('could not set sharing settings for template')({
-                                error,
-                                eventFilterData,
-                                templateId: template.id,
-                            }),
-                        );
-                    }))
+                .update(`eventFilters/${id}`, eventFilterData)
```

Comment:
Hey, 
Can `mutate` here be used here to avoid the usage of `getApi`? Just like it is done in the [original PR](https://github.com/dhis2/capture-app/pull/1514/files#diff-167426afa724a37993f1f3d242f2da27890060a7b6edc91320f907a27dcdb36aR84)
Thanks

Reply:
hey mutate is not available in v35 and v34. thanks!